Makefile

```diff
@@ -29,6 +29,10 @@ pdf:
 	cd bookdown ;\
 	Rscript -e 'bookdown::render_book(input = "index.Rmd", "bookdown::pdf_book", envir = new.env())'
 
+pinp:
```

Comment:
Can we please first focus on a single pdf output format? I don't see why we would want to use pinp, it is clearly not made for books.

Reply:
This can be removed, I just thought it make be handy to call this manually from time to time..

Reply:
Removed.